### PR TITLE
Get tenancies by ref should trim character field

### PR DESCRIPTION
**Pull request by LBHRShetty**

### CODE REVIEW

File: LBHTenancyAPI/Gateways/StubTenanciesGateway.cs [hunk 1]
```diff
@@ -0,0 +1,31 @@
+﻿using System.Collections.Generic;
+using LBHTenancyAPI.Domain;
+
+namespace LBHTenancyAPI.Gateways
+{
+    public class StubTenanciesGateway : ITenanciesGateway
+    {
+        private Dictionary<string, TenancyListItem> StoredTenancyListItems;
```
Comment (VladyslavAtamanyuk) [inline LBHTenancyAPI/Gateways/StubTenanciesGateway.cs]: Not sure about this but if this is a private list do we still add a "_" prefix to it?

File: LBHTenancyAPITest/Test/Controllers/TenanciesTest.cs [hunk 2]
```diff
@@ -11,18 +13,198 @@ namespace LBHTenancyAPITest.Test.Controllers
...
+        }
+
+        private static string ResponseJson(ObjectResult response)
+        {
+            return JsonConvert.SerializeObject(response.Value);
+        }
+
+        private class ListTenanciesSpy : IListTenancies
+        {
+            private readonly List<object> calledWith;
```
Comment (VladyslavAtamanyuk) [inline LBHTenancyAPITest/Test/Controllers/TenanciesTest.cs]: Need to prefix this with "_"

File: LBHTenancyAPITest/Test/Controllers/TenanciesTest.cs [hunk 3]
```diff
@@ -11,18 +13,198 @@ namespace LBHTenancyAPITest.Test.Controllers
...
+
+            public void AssertCalledWith(List<object> expectedArguments)
+            {
+                Assert.Equal(expectedArguments, calledWith[0]);
+            }
+        }
+
+        private class ListTenanciesStub : IListTenancies
+        {
+            private readonly Dictionary<string, ListTenancies.ResponseTenancy> stubTenancies;
```
Comment (VladyslavAtamanyuk) [inline LBHTenancyAPITest/Test/Controllers/TenanciesTest.cs]: More prefixing here too

File: LBHTenancyAPITest/Test/Gateways/UhTenanciesGatewayTest.cs [hunk 4]
```diff
@@ -478,5 +482,12 @@ private void InsertArrearsActions(string tenancyRef, string actionCode, DateTime
 
             command.ExecuteNonQuery();
         }
+
+        public string TrimmedValue(string passedValue)
```
[the remark this answers is not in the document]
Comment (LBHRShetty) [reply LBHTenancyAPITest/Test/Gateways/UhTenanciesGatewayTest.cs]: done!

**Pull request merged**